ci-scripts/unit-tests-core-lib.sh

```diff
@@ -1,130 +1,124 @@
 #!/usr/bin/env bash
-set -e
 set -o pipefail
 
+# -azure argument used by azure hyperspace pipeline
+if [[ $1 == '-azure' ]]; then
+    target=""
+else
+    set -e
+    exec 5>&1
+    target=" | tee /dev/fd/5"
+fi
+
 echo "-----"
 
 echo "Running unit tests and code coverage for core"
-exec 5>&1
-output=$(ng test core --watch=false --sourceMap --code-coverage --browsers=ChromeHeadless | tee /dev/fd/5)
+
+output=$(eval "ng test core --watch=false --sourceMap --code-coverage --browsers=ChromeHeadless"$target)
```

Comment:
The only way to have the pipe '|' concat to a string then being interpreted as a command, is to use 'eval' afterwards.